### ISSUE TITLE
TypeError on symmetrical many-to-many fields

### ISSUE THREAD

**Issue by btmills** [charettes/django-seal#50]
Thanks for this very helpful tool! For now we're able to work around this by changing `SealableModel` back to `models.Model` on that one particular model and not sealing those queries. This is so useful though that I'm happy to help if there's any additional debugging I can do.

- Python 3.8.6 (via Docker)
- Django 3.1.3
- django-seal 1.2.3

```py
# models.py
from seal.models import SealableModel

class SymmetricalModel(SealableModel):
    symmetrical = models.ManyToManyField("self")
```

If I run any manage command, even `python manage.py --help`, I get the following stack trace:

```
Traceback (most recent call last):
  File "manage.py", line 8, in <module>
    django.setup()
  File "/usr/local/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python3.8/site-packages/django/apps/registry.py", line 114, in populate
    app_config.import_models()
  File "/usr/local/lib/python3.8/site-packages/django/apps/config.py", line 211, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/src/herbie/contacts/models.py", line 2281, in <module>
    class TestModel(SealableModel):
  File "/usr/local/lib/python3.8/site-packages/django/db/models/base.py", line 321, in __new__
    new_class._meta.apps.register_model(new_class._meta.app_label, new_class)
  File "/usr/local/lib/python3.8/site-packages/django/apps/registry.py", line 232, in register_model
    self.do_pending_operations(model)
  File "/usr/local/lib/python3.8/site-packages/django/apps/registry.py", line 425, in do_pending_operations
    function(model)
  File "/usr/local/lib/python3.8/site-packages/django/apps/registry.py", line 404, in apply_next_model
    self.lazy_model_operation(next_function, *more_models)
  File "/usr/local/lib/python3.8/site-packages/django/apps/registry.py", line 416, in lazy_model_operation
    apply_next_model(model_class)
  File "/usr/local/lib/python3.8/site-packages/django/apps/registry.py", line 404, in apply_next_model
    self.lazy_model_operation(next_function, *more_models)
  File "/usr/local/lib/python3.8/site-packages/django/apps/registry.py", line 390, in lazy_model_operation
    function()
  File "/usr/local/lib/python3.8/site-packages/seal/models.py", line 75, in make_remote_field_descriptor_sealable
    make_descriptor_sealable(related_model, accessor_name)
  File "/usr/local/lib/python3.8/site-packages/seal/models.py", line 59, in make_descriptor_sealable
    descriptor = getattr(model, attname)
TypeError: getattr(): attribute name must be string
```
The [docs on symmetrical many-to-many relationships](https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.ManyToManyField.symmetrical) have this note:

> If you do not want symmetry in many-to-many relationships with self, set symmetrical to False. This will force Django to add the descriptor for the reverse relationship, allowing ManyToManyField relationships to be non-symmetrical.

Based on the stack trace ending in `make_descriptor_sealable`, perhaps it's assuming the existence of the opposite descriptor that Django isn't generating?

**Comment by charettes** [comment]
Thanks for the detailed report @btmills I'll have a look at it by EoD and get back to you as soon as I've got more details.

**Comment by charettes** [comment]
Fixed in the 1.3.0 release.

**Comment by btmills** [comment]
Thank you, that was fast! I've upgraded to 1.3.0 and confirmed the fix works :tada:
